typescript/web/src/components/spinner/spinner.tsx

```diff
@@ -1,11 +1,15 @@
-import { Spinner as ChakraSpinner, SpinnerProps } from "@chakra-ui/react";
+import {
+  Spinner as ChakraSpinner,
+  SpinnerProps,
+  useColorModeValue as mode,
+} from "@chakra-ui/react";
 import React from "react";
 
 export const Spinner = (props: SpinnerProps) => {
   return (
     <ChakraSpinner
       aria-label="loading indicator"
-      emptyColor="gray.200"
+      emptyColor={mode("gray.200", "gray.800")}
```

Comment:
Thanks for fixing the spinner in the Dark theme ❤️
To continue our discussion; in a "non-pragmatic world", it should even have been included in a separate PR 👍 It's just not always worth the additional effort as long as it's in an individual commit.
(_you can self-resolve this comment once read_ 😉 )